### PR TITLE
[TW-1119] VSC Add collections and environments

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Add bullet point about collections and environments
* Updated screenshot
* Added blog post URL to right sidebar (**Note**: The blog post isn't live yet.)

### CODE REVIEW

**Review by nerminamiller-postman — APPROVED** [review]
LGTM!

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by pankajk07 — APPROVED** [review]

File: src/pages/docs/getting-started/about-vs-code-extension.md [hunk 1]
```diff
@@ -32,4 +32,4 @@ The Postman VS Code extension enables you to develop and test your APIs in Postm
 
 To install and learn more about the VS Code extension, visit the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Postman.postman-for-vscode).
 
-![Postman VS Code extension](https://assets.postman.com/postman-docs/postman-vs-code-extension.jpg)
+![Postman VS Code extension](https://assets.postman.com/postman-docs/postman-vs-code-extension-2.jpg)
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/about-vs-code-extension.md:35]: New screenshot looks good!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**